front/styles/global.css

```diff
@@ -11,3 +11,17 @@ main {
 #__next {
   @apply h-full;
 }
+
+.loading-dots::after {
```

Comment:
not sure that's where to put it -- and at this time just a suggestion

Reply:
that's a fair place